### PR TITLE
Dodanie wsparcia dla setTrackingNumber w kliencie PublicApi

### PR DESCRIPTION
**Pull request by jasufo**

### CODE REVIEW

File: src/Linker/Api/Client/HttpApiClient.php [hunk 1]
```diff
@@ -3,6 +3,7 @@
 namespace Linker\Api\Client;
 
 use GuzzleHttp\Client;
+use http\Env\Request;
```
Comment (hrba) [inline src/Linker/Api/Client/HttpApiClient.php]: Wrong import

File: src/Linker/Api/Client/HttpApiClient.php [hunk 2]
```diff
@@ -143,4 +145,25 @@ public function updateOrder($id, OrderInterface $order)
...
+            'body'    => $content
+        ];
+        echo $content;
+        try {
+            return $this->client->request('PUT', $endpoint, $options);
+        } catch (ServerException $e) {
+            throw new ApiException($e->getResponse()->getReasonPhrase(), $e->getResponse()->getStatusCode(), $e);
+        }
+    }
 }
```
Comment (hrba) [inline src/Linker/Api/Client/HttpApiClient.php]: No new line at end of file

File: src/Linker/Api/LinkerClientInterface.php [hunk 3]
```diff
@@ -3,7 +3,9 @@
 namespace Linker\Api;
 
 
+use http\Env\Request;
```
Comment (hrba) [inline src/Linker/Api/LinkerClientInterface.php]: Wrong import

**Review by hrba — CHANGES_REQUESTED** [review]

File: src/Linker/Api/Client/HttpApiClient.php [hunk 4]
```diff
@@ -143,4 +145,25 @@ public function updateOrder($id, OrderInterface $order)
...
+     */
+    public function setTrackingNumber($id, TrackingNumber $trackingNumber)
+    {
+        $endpoint = $this->endpoint . '/orders/' . $id . '/trackingnumber?apikey=' . $this->apiKey;
+        $content  = $this->serializer->serialize($trackingNumber, 'json');
+        $options  = [
+            'headers' => $this->headers,
+            'body'    => $content
+        ];
+        echo $content;
```
Comment (michalploneczka) [inline src/Linker/Api/Client/HttpApiClient.php]: echo.... :)

**Review by hrba — APPROVED** [review]

**Pull request merged**